### ISSUE TITLE
gluon-status-page, gluon-mesh-babel: extend statistics for babel respondd module

### ISSUE THREAD

**Issue by christf** [freifunk-gluon/gluon#1054]
these resources remain unimplemented in the gluon-mesh-babel respondd provider:

- [x] nodeinfo.software.babeld
- [x] statistics.clients
- [x] statistics.traffic
- [ ] statistics.gateway
- [ ] statistics.gateway_nexthop
- [x] neighbours.*

**Comment by christf** [comment]
@NeoRaider how do you feel about turning statistics.gateway into a list contianing the fields ipaddress, txcost and rxcost? I would rather not put the additional two fields for the cost on the same level like gateway itself as they correspond to the gateway only.

**Comment by NeoRaider** [comment]
Do we actually know the IP address of the gateway, and not just the nexthop?

Changing this sounds like a good idea, but for consistency, adding the metric (TQ) to the gateway field for batman-adv would make sense, too. I don't think we can do that though without breaking compatiblity unless we introduce a new name for the "gateway" field. Something like "default_route" might make sense.

**Comment by christf** [comment]
I had a bad and wrong implementation for obtaining the gateway IP address as I just noticed. 
We do know the fe80-address of gateway_nexthop which I just included in the implementation (build & test still has to happen)

What do we do about the gateway then? I cannot see a field from the babel data from which we could derive that. A traceroute might reveal that. But do we actually want respondd to run traceroute to determine that information?

still it would be good to know which gateway was used....
